composer.json

```diff
@@ -20,7 +20,8 @@
         }
     ],
     "require": {
-        "php": ">=5.5.0"
+        "php": ">=5.5.0",
+        "detailnet/dfw-normalization": "~0.1.6"
```

Comment:
Don't use such a strict version. This will give us problems because `dfw-normalization` is used by multiple other dependencies of a project.

Maybe just suggest the usage of `dfw-normalization` so that the project can decide on it's own which version to use.